configuration/exodus-pipeline.yaml

```
@@ -0,0 +1,297 @@
+    prod:
+      object-key: build-stage/exodus-lambda-pkg.yaml
+
+Resources:
+  SNSTopic:
+    Type: AWS::SNS::Topic
+    Properties:
+      TopicName: !Sub exodus-sns-${env}
+      Subscription:
+        - Endpoint: !Ref email
+          Protocol: email
+
+  CodePipelineNotification:
+    Type: AWS::CodeStarNotifications::NotificationRule
+    Properties:
+      Name: !Sub exodus-pipeline-${env}
+      DetailType: FULL
+      EventTypeIds:
+        - codepipeline-pipeline-pipeline-execution-failed
+      Resource: !GetAtt CodePipeline.Arn
+      Targets:
+        - TargetType: SNS
+          TargetAddress: !GetAtt SNSTopic.Arn
+
+  CodePipeline:
+    Type: AWS::CodePipeline::Pipeline
+    Properties:
+      Name: !Sub exodus-pipeline-${env}
+      ArtifactStore:
+        Type: S3
+        Location: !Ref CodePipelineBucket
+      RoleArn: !GetAtt CodePipelineRole.Arn
+      Stages:
+        - !If
+          # Create Github source for dev pipeline
+          - !Equals [!Ref env, dev]
+          - Name: Source
+            Actions:
+              - Name: Source
+                ActionTypeId:
+                  Category: Source
+                  Owner: ThirdParty
+                  Provider: Github
+                  Version: 1
+                Configuration:
+                  Owner: release-engineering
+                  Repo: exodus-lambda
+                  Branch: deploy
+                  OAuthToken: !Ref github-token
+                  PollForSourceChanges: false
+                OutputArtifacts:
+                  - Name: SourceArtifact
+          # Create S3 source for stage, prod pipelines
+          - !Equals [!Ref env, prod]
+          - Name: Source
+            Actions:
+              - Name: Source
+                ActionTypeId:
+                  Category: Source
+                  Owner: AWS
+                  Provider: S3
+                  Version: 1
+                Region: !Ref region
+                Configuration:
+                  S3Bucket: exodus-pipeline-artifacts
+                  S3ObjectKey: !FindInMap [SourceMap, !Ref env, object-key]
+                  PollForSourceChanges: false
+                OutputArtifacts:
+                  - Name: SourceArtifact
+        # Create build stage for dev pipeline
+        - !If
+          - !Equals [!Ref env, dev]
+          - Name: Build
+            Actions:
+                Name: Build
+                ActionTypeId:
+                  Category: Build
+                  Owner: AWS
+                  Provider: CodeBuild
+                  Version: 1
+                Region: !Ref region
+                Configuration:
+                  EnvironmentVariables:
+                    - Name: ENV_TYPE
+                      Value: !Ref env
+                      Type: PLAINTEXT
+                    - Name: AWS_REGION
+                      Value: !Ref region
+                      Type: PLAINTEXT
+                  ProjectName: exodus-lambda
```

The ProjectName referenced in the CodePipeline conflicts with the CodeBuild project name defined in line 205:
```suggestion
                  ProjectName: exodus-codebuild
```

---

Thanks!